conjure-java-core/src/integrationInput/java/com/palantir/product/MultipleOrderedStages.java

```diff
@@ -180,7 +180,7 @@ public static ItemStageBuilder builder() {
 
     @Generated("com.palantir.conjure.java.types.BeanBuilderGenerator")
     @JsonIgnoreProperties(ignoreUnknown = true)
-    private static final class DefaultBuilder implements Builder {
+    static final class DefaultBuilder implements Builder {
```

Comment:
unfortunately when this is private, ti cannot be referenced bythe `JsonDeserialize` annotation.